guides/development-workflow.md

```diff
@@ -0,0 +1,341 @@
+# Development workflow
+
+## Introduction
+
+<!-- Short intro. Link to playbook for the details -->
+
+### Development principles
+
+<!-- Explain where this principles came from and how they change -->
+
+#### Communicate frequently
+
+  - space to learn from each other
+  - make shared decisions
+  - prevent duplication
+  - solve problems quicker
+
+#### Two pairs of eyes
+
+All code we write needs be reviewed both another developer. We hope that doing this will mean we:
+  - ship better quality code
+  - get the opportunity to learn new skills
+  - increase the shared knowledge of our products
+
+#### Everything is tracked
+
+All code we write is represented in Trello so that:
+  - the client can easily visualise how we're doing and what needs their attention
+  - the team can see and react to bottle necks
+  - delivery managers can see what cards have been lingering
+  - developers can be confident what they are deploying
+
+#### Own your code
+
+Having the opportunity to see a piece of work through to completion is a great way to for developers to grow. To encourage this we ask reviewers to leave their feedback, ask questions and discuss the code but not to make any changes without the okay from the author. This will hopefully allow us to:
+
+  - learn from our mistakes
+  - create personal achievements from seeing work through to production
+
+#### Do the smallest amount of good work
+
+We aspire to regularly ship [good code](https://robots.thoughtbot.com/what-is-good-code) and delivery it to the users as often as possible. Doing so  efficiently requires a balanced approach to development.
+
+Good signs:
+- the code is easy to read and understand
+- the code didn't take too long to write
+- the code is relatively easy to change
+
+Bad signs:
+- spending too much time over engineering (gold plating) our code
+- delivering too quickly at the cost of any code quality
```

Comment:
I think we should talk about this in the next dev chat tomorrow? I had a conversation in one of the retros this week about whether some of us are doing too thorough code reviews and what standard we should be working towards (so we are maintaining good quality code but not impeding delivery)